code/cgame/cg_players.c

```diff
@@ -2531,6 +2536,13 @@ void CG_Player(centity_t *cent) {
 	else
 		ci->ftIsFrozen = qfalse;
 
+	// push killerduck state into clientinfo
+	// we need that for the scorebord
+	if (CG_CatchTheKillerduck() && CG_IsKillerDuck)
```

Comment:
this is wrong - as the function pointer is checked for not being null here - which is always true - should be `CG_IsKillerDuck()`

The check is redundant because the `EF_KILLERDUCK` flag is only set in killerduck game mode